third_party/BUILD

```diff
@@ -13,3 +13,15 @@ cc_library(
         ],
     }),
 )
+
+# portaudio
+cc_library(
```

Comment:
Don't follow the bad practice of define `cc_library` targets with linkopts only.
See `third_party/xxx/workspace.bzl` and `third_party/xxx/xxx.BUILD` on how to configure `bazel` external dependency.

Reply:
Thanks I will. Btw the doc [how_to_add_an_external_dependency.md](https://github.com/ApolloAuto/apollo/blob/master/docs/howto/how_to_add_an_external_dependency.md) may be updated later.

Reply:
> Thanks I will. Btw the doc [how_to_add_an_external_dependency.md](https://github.com/ApolloAuto/apollo/blob/master/docs/howto/how_to_add_an_external_dependency.md) may be updated later.

OK.